pkg/helper/helpers.go

```diff
@@ -68,3 +70,10 @@ func MaxLabelLength(in string) string {
 	}
 	return in[:maxLabelLength]
 }
+
+func UnstructuredDuration(in *metav1.Duration) string {
```

Comment:
pls add ut coverage for this api, I have one for u :)
```golang
func TestUnstructuredDuration(t *testing.T) {

  t.Run("nil input", func(t *testing.T) {
    var d *metav1.Duration
    got := UnstructuredDuration(d)
    want := ""
    
    if got != want {
      t.Errorf("got %q, want %q", got, want) 
    }
  })

  t.Run("valid input", func(t *testing.T) {
    d := &metav1.Duration{Duration: time.Minute}
    want := "1m0s"
    
    got := UnstructuredDuration(d)
    
    if got != want {
      t.Errorf("got %q, want %q", got, want)
    }
  })

}
```

Reply:
thanks! done